boards/UNTESTED_kgpe-d16_server-whiptail/UNTESTED_kgpe-d16_server-whiptail.config

```diff
@@ -31,6 +31,8 @@ CONFIG_GPG2=y
 CONFIG_KEXEC=y
 CONFIG_UTIL_LINUX=y
 CONFIG_LVM2=y
+CONFIG_E2FSPROGS=y
+CONFIG_EXFATPROGS=y
```

Comment:
Nearly all boards got this - only exceptions I see are the "legacy-flash" boards and talos-II?

Could we default these to 'y' and set them to 'n' only in the few boards that should _not_ get them?

Reply:
Will do that.

Talos had EFAT staging activated under 5.5 kernel from above commit. We will see if that works as expected.
I would love to have exfat under legacy-flash boards as well, which EXFAT driver was included per prior commits. Let's see if they build with the *PROGS added by default.

From now on, those modules are included/compiled(musl) by default unless manually deactivated in board configs:
```
modules/busybox:2:CONFIG_BUSYBOX ?= y
modules/e2fsprogs:2:CONFIG_E2FSPROGS ?= y
modules/exfatprogs:2:CONFIG_EXFATPROGS ?= y
modules/musl-cross:1:CONFIG_MUSL ?= y
modules/zlib:1:CONFIG_ZLIB ?= y
```